builder/lifecycle_test.go

```diff
@@ -18,7 +18,8 @@ import (
 )
 
 func TestLifecycle(t *testing.T) {
-	color.NoColor = true
+	color.Disable(true)
+	defer func() { color.Disable(false) }()
```

Comment:
Probably don't need to re-enable it right? Unless I haven't looked further down yet and this modifies some local configuration or something.

Reply:
The color is enabled by default, so when I disable it I want to re-enable it when I'm done testing to restore it to its default state.  Is it needed strictly speaking, maybe not, but in keeping with principle of least surprise I'd like the color library to be in it's default state for any other tests that might use it.

Reply:
I see your point